open-machine-learning-jupyter-book/data-science/data-visualization/visualization-proportions.md

```diff
@@ -142,29 +144,31 @@ fig = plt.figure(
 
 Using a waffle chart, you can plainly see the proportions of cap colors of this mushrooms dataset. Interestingly, there are many green-capped mushrooms!
 
-✅ Pywaffle supports icons within the charts that use any icon available in [Font Awesome](https://fontawesome.com/). Do some experiments to create an even more interesting waffle chart using icons instead of squares.
+```{seealso}
+Pywaffle supports icons within the charts that use any icon available in [Font Awesome](https://fontawesome.com/). Do some experiments to create an even more interesting waffle chart using icons instead of squares.
+```
 
 In this lesson, you learned three ways to visualize proportions. First, you need to group your data into categories and then decide which is the best way to display the data - pie, donut, or waffle. All are delicious and gratify the user with an instant snapshot of a dataset.
 
 ## Self study
 
 Sometimes it's not obvious when to use a pie, donut, or waffle chart. Here are some articles to read on this topic:
 
-https://www.beautiful.ai/blog/battle-of-the-charts-pie-chart-vs-donut-chart
+* a https://www.beautiful.ai/blog/battle-of-the-charts-pie-chart-vs-donut-chart
 
-https://medium.com/@hypsypops/pie-chart-vs-donut-chart-showdown-in-the-ring-5d24fd86a9ce
+* b https://medium.com/@hypsypops/pie-chart-vs-donut-chart-showdown-in-the-ring-5d24fd86a9ce
 
-https://www.mit.edu/~mbarker/formula1/f1help/11-ch-c6.htm
+* c https://www.mit.edu/~mbarker/formula1/f1help/11-ch-c6.htm
 
-https://medium.datadriveninvestor.com/data-visualization-done-the-right-way-with-tableau-waffle-chart-fdf2a19be402
+* d https://medium.datadriveninvestor.com/data-visualization-done-the-right-way-with-tableau-waffle-chart-fdf2a19be402
```

Comment:
remove abcd. And use URL preview instead of the URL text directly, such as [Battle of the Charts: Pie Chart vs. Donut Chart| The Beautiful Blog](https://www.beautiful.ai/blog/battle-of-the-charts-pie-chart-vs-donut-chart). Or you can create it from here, https://zbib.org/

Reply:
You can try paste url vscode plugin